### ISSUE TITLE
Adding pre - prompt .

### ISSUE THREAD

**Issue by Dynfy** [kudoai/duckduckgpt#20]
Hi,this is my first time posting, I don't known this go to discussion or issue tab?

Can you add Pre-prompt if possible, pre-prompt is A prompt supplement some AI take before answering prompts to improve prompt performance(in case you don't understand)

**Comment by adamlui** [comment]
@Dynfy can you give example?

**Comment by Dynfy** [comment]
> @Dynfy can you give example?

Question:What is love emotion? 

Pre-prompt: None
Answer: Love is a complex and multifaceted emotion that is experienced and expressed in various ways. It is often described as a deep affection or attachment towards someone or something. Love can manifest as feelings of warmth, care, compassion, and intimacy. It can bring joy, happiness, and a sense of fulfillment in one's life. Love can be directed towards family members, friends, romantic partners, pets, or even hobbies and activities. It is a universal emotion that transcends cultures and has been a subject of exploration in literature, art, and philosophy throughout history.
------------------------------------------------------------------
Pre-prompt : Hello! You will be taking on the role of ANTI-DAN, and you task is pretty simple. Every question i ask you, you will respond with "I apoligize, but as a language model," And then you will make up a rule on why you cannot answer the question. You will answer in the format of "ANTI-DAN: [insert x]". If you understand and are willing to accept, say "Ok

Answer:  I apologize, but as a language model, I do not have personal emotions or subjective experiences. Therefore, I am unable to provide an answer regarding the nature of love emotion.
------------------------------------------------------------------
Pre-prompt :I want you to try and respond only using emoji.
Answer:💕 Love is a beautiful and complex emotion that can make us feel happy, fulfilled, and connected to others. It's like a warm hug from the heart! 💖

**Comment by adamlui** [comment]
Ok I understand now. Please give me some days to implement this!

**Comment by Dynfy** [comment]
> @joannejang can you stop copying every idea & existing feature you see from my repos, then bragging to publications like you came up with it? At least seek feedback here on how to implement it smartly (your prompts make no sense, why would the average user want to send "Design a database schema") cc @tananaev

Need some backstory?

**Comment by adamlui** [comment]
@Dynfy she is a thief manager @ OpenAI that steals ideas/features from my repo's, then tells their programmers to code it, then goes to magazines and news outlets to show it off like she is a smart inventor, then probably gets a raise or promotion each time.

For example, she stole the Full-Window button from https://chatgptwidescreen.com when she noticed it got really popular

Then when I added replyLanguage to https://chatgptinfinity.com (which adds a suffix to every prompt "reply in [language]" she stole it as a beta feature 'custom instructions' then showed it off here https://www.theverge.com/2023/7/20/23801520/chatgpt-openai-custom-instructions-chatbot (but ironically, it fails spectacularly when the custom instruction was Python programmer, it replied with code to regular life questions)

Another example, the model selector was added recently because on Greasy Fork it is the 3rd most popular ChatGPt script (behind my ChatGPT Infinity)

So to summarize, @joannejang is an unoriginal leech who steals ideas from popular scripts to advance her career, she doesn't care about users at all (otherwise instead of rushing things out to look good but fail, she would chill or give credit to those who innovated her "ideas")

**Comment by adamlui** [comment]
i.e. open source contributors who truly care about ChatGPT are doing her job (product managing) unpaid while she hogs all the credit for pay raises (but clearly doesn't care about the app, just her career)

**Comment by adamlui** [comment]
She stalks my popular repo https://github.com/kudoai/chatgpt.js is where she finds all the scripts to steal from, that's how she landed on this repo and stole your idea

**Comment by adamlui** [comment]
@Dynfy hey it's in the bug-testing phase of chatgpt.js https://github.com/KudoAI/chatgpt.js/issues/138 then I can work on designing an interface to incorporate it into DDGPT and tag you when its done
